modules/indexer/issues/bleve/bleve.go

```diff
@@ -156,12 +156,19 @@ func (b *Indexer) Search(ctx context.Context, options *internal.SearchOptions) (
 	var queries []query.Query
 
 	if options.Keyword != "" {
-		keywordQueries := []query.Query{
-			inner_bleve.MatchPhraseQuery(options.Keyword, "title", issueIndexerAnalyzer),
-			inner_bleve.MatchPhraseQuery(options.Keyword, "content", issueIndexerAnalyzer),
-			inner_bleve.MatchPhraseQuery(options.Keyword, "comments", issueIndexerAnalyzer),
+		if options.IsFuzzyKeyword {
```

Comment:
For keywords, it should be always fuzzy, how does a prefix search work for the issue/comment title/content search, this is totally not available.

Reply:
@lunny see https://github.com/go-gitea/gitea/pull/29706